src/dnslookup.c

```diff
@@ -606,7 +606,7 @@ struct SOA {
 	uint32_t dnssoa_minttl;
 };
 
-typedef void query_soa_fn(struct dns_ctx *ctx, struct SOA *result, void *data);
+typedef void query_soa_fn (struct dns_ctx *ctx, struct SOA *result, void *data);
```

Comment:
It doesn't seem consistent because there is no space between function name and arguments.